CONTRIBUTING.md

```diff
@@ -0,0 +1,92 @@
+# Contributing new schemes to PQClean
```

Comment:
Cool! Does it mean, it's OK to submit SIDH now?

Reply:
There are some things still moving; but we hope that we are in a shape to accept submissions soon. Notably, I also need to solve #1